File: Ileya/Day 1/Python/Level 2/task_1.py
def process_duplicates(arr):
    output = []
    range_ = len(arr)
    for index in range(range_):
        duplicate = False
        for instance in range (index):
            if arr[index] == arr[instance]:
                if duplicate == True:
                    duplicate = False
                    break
                duplicate = True
                
        if duplicate == True:
            output.append(arr[index])

    return output

File: Ileya/Day 1/JavaScript/Level 2/Task1.js
function processDuplicates(arr){
    let output = []
    for (let index = 0; index < arr.length; index++){
        let duplicate = false;
        for (let instance = 0; instance < index; instance++){
            if (arr[index] == arr[instance]){
                if (duplicate == true) {
                    duplicate = false
                    break
                }

                duplicate = true
            }
        }
                
        if (duplicate == true) {
            output.push(arr[index]);
        }
    }

    return output
}

module.exports = processDuplicates;
